versionutils/version.go

```diff
@@ -125,17 +130,26 @@ func (v Version) IsGreaterThan(lesser Version) (bool, bool) {
 		return false, true
 	}
 
-	if v.Label != lesser.Label {
+	if v.Label == lesser.Label {
+		if v.LabelVersion > lesser.LabelVersion {
+			return true, true
+		} else if v.LabelVersion < lesser.LabelVersion {
+			return false, true
+		}
 		return false, false
```

Comment:
I think this case is actually the 2 versions are entirely identical? (Which used to be the end of the function.) So it would be false, true?

Reply:
yeah! updated